### ISSUE TITLE
After restart, AS3 fails to start up due to failure with loading ATG Storage config

### ISSUE THREAD

**Issue by andreykashcheev** [F5Networks/f5-appsvcs-extension#451]
### Environment
 * Application Services Version: 3.27.0
 * BIG-IP Version: 15.1.0.4 - 0.0.6

### Summary
Please also include information about the reproducibility and the severity/impact of the issue.

When multiple extensions installed on BIGIP system (i.e. DO, TS, FAST and CFE), attempt to restart BIGIP system using "bigstart restart" command causes AS3 fail to start up and the following error can be seen in /var/log/restnode/restnode.log

```
Tue, 27 Apr 2021 23:26:18 GMT - severe: [appsvcs] {"message":"Failed to complete startup. Error: Command failed: tmsh -a load sys config merge file /tmp/__atg-storage_temp.conf\nData Input Error: Can't find the file, /tmp/__atg-storage_temp.conf\n\n at ChildProcess.exithandler (child_process.js:275:12)\n at emitTwo (events.js:126:13)\n at ChildProcess.emit (events.js:214:7)\n at maybeClose (internal/child_process.js:925:16)\n at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)","level":"error"}
```
It took me about ~10 attempts (i,e, install, restart, uninstall) before I was able to replicate the error. However,  In order to mitigate the socket hang-up problem "sleep 15" command needs to be added between installation calls

### Steps To Reproduce

 1) Install several extensions:
 
 ```
 curl -sk -u admin: -d '{"operation": "INSTALL","packageFilePath": "/var/lib/cloud/icontrollx_installs/f5-declarative-onboarding-1.20.0-2.noarch.rpm"}' http://localhost:8100/mgmt/shared/iapp/package-management-tasks
curl -sk -u admin: -d '{"operation": "INSTALL","packageFilePath": "/var/lib/cloud/icontrollx_installs/f5-cloud-failover-1.8.0-0.noarch.rpm"}' http://localhost:8100/mgmt/shared/iapp/package-management-tasks
curl -sk -u admin: -d '{"operation": "INSTALL","packageFilePath": "/var/lib/cloud/icontrollx_installs/f5-appsvcs-3.19.0-4.noarch.rpm"}' http://localhost:8100/mgmt/shared/iapp/package-management-tasks
curl -sk -u admin: -d '{"operation": "INSTALL","packageFilePath": "/var/lib/cloud/icontrollx_installs/f5-telemetry-1.19.0-3.noarch.rpm"}' http://localhost:8100/mgmt/shared/iapp/package-management-tasks
curl -sk -u admin: -d '{"operation": "INSTALL","packageFilePath": "/var/lib/cloud/icontrollx_installs/f5-appsvcs-templates-1.8.1-1.noarch.rpm"}' http://localhost:8100/mgmt/shared/iapp/package-management-tasks
 ```

 2) Hit info endpoint on each extension to confirm that it is up and running:
 
 ```
curl -s -u admin: http://localhost:8100/mgmt/shared/declarative-onboarding/info | jq .
curl -s -u admin: http://localhost:8100/mgmt/shared/cloud-failover/info | jq .
curl -s -u admin: http://localhost:8100/mgmt/shared/appsvcs/info | jq .
curl -s -u admin: http://localhost:8100/mgmt/shared/fast/info | jq .
curl -s -u admin: http://localhost:8100/mgmt/shared/telemetry/info | jq .
 ```

 3) Do BIGIP restart using "bigstart restart" command
 4) Check /var/log/restnoded/restnoded.log using the following command: 
```
egrep '(Startup Failed|Failed to complete|failed to register)' /var/log/restnoded/restnoded.log
```

Here is script which I used for replicating this issue:
https://ak-f5-cft.s3-us-west-2.amazonaws.com/validate_lx_extension_installation.sh

### Expected Behavior

After BIGIP restart, AS3 is up and running as expected

### Actual Behavior
A clear and concise description of what actually happens.
Please include any applicable error output.

After BIGIP restart, AS3 fails to start up and error, regarding ATG Storage, can be seen under restnoded log.

**Comment by dstokesf5** [comment]
Thank you for your feedback. This has been added to our internal product backlog as AUTOTOOL-2468.

**Comment by dstokesf5** [comment]
This issue was resolved as part of the 3.28.0 release.
